KsApi/MockService.swift

```diff
@@ -134,8 +134,11 @@
 
     fileprivate let incrementVideoStartError: ErrorEnvelope?
 
-    fileprivate let postCommentResponse: DeprecatedComment?
-    fileprivate let postCommentError: ErrorEnvelope?
+    fileprivate let deprecatedPostCommentResponse: DeprecatedComment?
+    fileprivate let deprecatedPostCommentError: ErrorEnvelope?
+
+    fileprivate let postCommentResponse: GraphMutationPostCommentResponseEnvelope?
```

Comment:
However since my later suggestion to only return `Comment` it would just be:

```swift
let postCommentResult: Result<Comment, ErrorEnvelope>?
```

Reply:
Ah I see. Made these updates with one difference: `GraphError` instead of `ErrorEnvelope` because we calling `applyMutation` which uses the `GraphError` type.

Reply:
There's a workaround for that 😄 
https://github.com/kickstarter/ios-oss/blob/master/KsApi/Service.swift#L179

Reply:
Thanks, updated.